### PR TITLE
fix: add CertificateChainMilan

### PR DESCRIPTION
**Pull request by haraldh**

### CODE REVIEW

File: src/attestation/sev.rs [hunk 1]
```diff
@@ -30,16 +30,21 @@ pub struct Measurement {
 #[derive(Debug, Eq, PartialEq, Deserialize, Serialize)]
 #[serde(rename_all = "kebab-case")]
 pub enum Message {
-    /// The Naples certificate chain carries the SEV platform
-    /// certificate chain, but the AMD ARK and ASK are "small"
+    /// The Naples certificate chain carries a SEV platform
```
Comment (npmccallum) [inline src/attestation/sev.rs]: Nitpick: `a` => `an`

File: src/attestation/sev.rs [hunk 2]
```diff
@@ -30,16 +30,21 @@ pub struct Measurement {
...
+    /// certificate chain, with the AMD ARK and ASK as "small"
     /// CA certificates (using 256 byte components).
     CertificateChainNaples(Chain),
 
-    /// The Rome certificate chain also carries the SEV platform
-    /// certificate chain, but the AMD ARK and ASK are "large"
+    /// The Rome certificate chain carries a SEV platform
```
Comment (npmccallum) [inline src/attestation/sev.rs]: Nitpick: `a` => `an`

**Review by npmccallum — CHANGES_REQUESTED** [review]

File: src/attestation/sev.rs (hunk 1 above)
Comment (haraldh) [reply src/attestation/sev.rs]: fixed

File: src/attestation/sev.rs (hunk 2 above)
Comment (haraldh) [reply src/attestation/sev.rs]: fixed

**Review by npmccallum — APPROVED** [review]

**Pull request merged**